### PR TITLE
Compute complements vegetal

### PR DESCRIPTION
**Pull request by paulboosz**
[User story](https://www.notion.so/Pr-parer-l-int-gration-des-compl-ments-pour-les-ingr-dients-v-g-taux-7378f2060db74c18a39258c9fccc7d73?pvs=4)

### CODE REVIEW

File: public/data/food/processes.json
Comment (paulboosz) [inline public/data/food/processes.json:1]: the organic beef bug is still here...

File: public/data/food/ingredients.json [hunk 1]
```diff
@@ -86,14 +92,17 @@
     "density": 1.0,
     "inedible_part": 0,
     "transport_cooling": "none",
-    "visible": true,
     "ecosystemicServices": {
-      "hedges": 0.33,
-      "plotSize": 0.45,
+      "hedges": 0.3315,
+      "plotSize": 0.44625,
```
Comment (paulboosz) [inline public/data/food/ingredients.json:97]: computed ecosystemicServices = previous ecosystemicServices from gsheet :)

**Review by ccomb — APPROVED** [review]
Nice shot! I've no comment

**Comment by ccomb** [comment]
Actually there is just a conflict to resolve

**Pull request merged**